lib/nsxv.py

```diff
@@ -326,14 +329,25 @@ def __str__(self):
     for s in services:
       service = '%s%s' % (service, s)
 
+    # applied_to
+    applied_to_list = ''
+    if self.applied_to:
+      applied_to_list = '<appliedToList>'
+      applied_to_element = '%s%s%s' % (_XML_TABLE.get('appliedToStart'),
+                              self.applied_to,
```

Comment:
Thanks for fixing this one but the indentation is wrong, You need to align it by adding 9 spaces.